extensions/libarchive/MergeContent.cpp

```diff
@@ -246,27 +247,30 @@ bool MergeContent::processBin(core::ProcessContext *context, core::ProcessSessio
     mimeType = "application/zip";
   } else {
     logger_->log_error("Merge format not supported %s", mergeFormat_);
+    session.remove(merge_flow);
     return false;
   }
 
   std::shared_ptr<core::FlowFile> mergeFlow;
```

Comment:
nitpicking and old code, but this seems unused
```suggestion
```

Reply:
Removed in 8a269048b9b6f451f0baedd584a4358286908a60